specification/includes/email.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "string",
+  "example": "john@doe.com",
+  "pattern": "^(([^<>()\\[\\]\\.,;:\\s@\"]+(\\.[^<>()\\[\\]\\.,;:\\s@\"]+)*)|(\".+\"))@((\\[[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}])|(([a-zA-Z\\-0-9]+\\.)+[a-zA-Z]{2,}))$"
```

Comment:
I got this pattern from http://emailregex.com/
This should work for almost all emails, but if we want a less strict pattern, we could also change it to something like:
`"^(.)+@(.)+\.(.)+$"`

Reply:
(.)+(.)+(.) = 3 8008's